test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,17 @@
 
 public class UtilsTest {
 
+    @Test
+    public void elementsContainNull() throws Exception {
```

Comment:
It is better to follow the convention that the names of the method being tested and the test method are the same, at least for the sake of better readability and tracking.